spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/test/domain/SubTradeRepository.java

```diff
@@ -44,4 +44,8 @@
 			+ " WHERE id = @id AND trader_id = @trader_id")
 	List<SubTrade> getList(@Param("id") String id, @Param("trader_id") String tradeId, Sort sort);
 
+	@Query("SELECT count(1) FROM :org.springframework.cloud.gcp.data.spanner.test.domain.SubTrade:"
```

Comment:
please test that the final sql string is what we expect

Reply:
added.
I found that two tests were absolutely identical - noPageableParamNotWrappedQueryTest & noPageableParamWrappedQueryTest.
So I left only one.